swap_meet/vendor.py

```diff
@@ -1,2 +1,73 @@
+# from .item import Item
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
+        self.inventory = [] if inventory is None else inventory
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item_to_remove):
+        if item_to_remove in self.inventory:
+            self.inventory.remove(item_to_remove)
+            return item_to_remove
+        return False
```

Comment:
This works! However, I'd prefer the logic to be flipped and that the guard clause pattern is used here so you check first for invalid input and then return `False`. 

As the method is currently written, it seems like your main logic is to return `False`, but the main logic that should be emphasized is that the method removes an item and returns it.